apps/frontend/components/shadcn/pagination.tsx

```diff
@@ -50,10 +50,8 @@ const PaginationButton = ({
       type="button"
       className={cn(
         buttonVariants({ variant: 'ghost', size }),
-        'text-base',
-        isActive
-          ? 'text-primary-strong hover:text-primary-light hover:bg-gray-100'
-          : 'text-[#8A8A8A] hover:text-slate-400',
+        'text-base, align-middle',
+        isActive ? 'text-primary-strong' : '',
```

Comment:
```suggestion
        isActive && 'text-primary-strong',
```
이렇게 쓰면 불필요한 빈 문자열을 반환하지 않고 간결해서 선호합니다.
하지만 성능 상 둘의 차이는 거의 없다고 하는데
어떻게 생각하시는지 궁금해요.